### DISCUSSION TITLE
How does the macOS cursor acceleration/speed correlate to Windows cursor acceleration/speed?

### DISCUSSION THREAD

**Discussion by VincentBounce** [linearmouse/linearmouse#295] [Q&A]
(Still exist a topic about speed, but not about acceleration).
The cursor is accelerated when you check "Enhance pointer precision" on Windows 10-11. This is the default setting.
What is the exact value of **acceleration** to set on the LinearMouse app to get precisely the same behavior on macOS with the same mouse hardware? This value can be placed from 0 to 20, not only a binary parameter yes/no like Windows.

Same question for **speed**: what is the value of the parameter between 0 and 1 to get the same speed as the Windows 10-11 default speed?

Below are the default Mouse Properties on Windows 10-11:
![image](https://user-images.githubusercontent.com/64386272/211142998-a061673b-1a2c-4b9f-8c8e-1ff9d2c23e86.png)

Below is an example of settings on LinearMouse
![image](https://user-images.githubusercontent.com/64386272/211143019-77185e6a-9250-4aa5-af3c-515ddd7c6919.png)

Below are the Windows 10-11 Mouse default settings on the Registry Editor:
![image](https://user-images.githubusercontent.com/64386272/211143047-7585d28a-9875-471d-8393-cc634f51df62.png)

I appreciate your help.

**Comment by carl2x** [comment]
https://github.com/linearmouse/linearmouse/discussions/121
